rules/sqlperf-verify-indexes-used/rule.md

```diff
@@ -0,0 +1,54 @@
+---
+seoDescription: Verify that your SQL indexes are being used
+type: rule
+archivedreason:
+title: Do you know how to verify that your SQL indexes are being used?
+guid: 0df7ba89-a104-4668-b6f0-4ac9e77bbb96
+uri: sqlperf-verify-indexes-used
+created: 2024-07-23T07:26:54.0000000Z
+authors:
+  - title: Bryden Oliver
+    url: https://ssw.com.au/people/bryden-oliver
+related:
+  - sqlperf-reduce-table-size
+  - sqlperf-select-required-columns
+  - sqlperf-avoid-implicit-type-conversions
+  - sqlperf-avoid-looping
+  - sqlperf-use-and-instead-of-or
+  - sqlperf-minimise-large-writes  
+---
+
+So you've created some indexes on your database tables. How can you tell if they are being used by your queries?
+
+<!--endintro-->
+
+You'd be surprised to find out how often the indexes that have been defined on a table are not being used. As such it's worth checking that your indexes are being used.
+If you know the exact query being run, you can simply run the query in SQL Server Management Studio or Azure Data Studio with "Include Actual Execution Plan" turned on. Then check the exxecution plan to verify what indexes were used.
```

Comment:
Hi @brydeno 

1. I believe we could add a paragraph break here... possibly with a heading "Ways to verify"
2. Fix the typo "exxecution"

Reply:
Done